### ISSUE TITLE
Mtl file does not exist

### ISSUE THREAD

**Issue by A6248384** [ParthaEth/GIF#14]
hello
when I run generate_random_samples.py I found this warning.Please tell me where I can find this file.Thanks
/usr/local/lib/python3.6/dist-packages/pytorch3d/io/obj_io.py:457: UserWarning: Mtl file does not exist: /content/GIF/GIF_resources/input_files/flame_resource/template.mtl
  warnings.warn(f"Mtl file does not exist: {f}")

**Comment by A6248384** [comment]
creating the FLAME Decoder
/content/GIF/my_utils/photometric_optimization/models/FLAME.py:81: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.register_buffer('dynamic_lmk_faces_idx', torch.tensor(lmk_embeddings['dynamic_lmk_faces_idx'], dtype=torch.long))
/content/GIF/my_utils/photometric_optimization/models/FLAME.py:82: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.register_buffer('dynamic_lmk_bary_coords', torch.tensor(lmk_embeddings['dynamic_lmk_bary_coords'], dtype=self.dtype))
tcmalloc: large alloc 1258291200 bytes == 0xba3b6000 @  0x7f297243e1e7 0x7f296febe41e 0x7f296ff0ebdb 0x7f296fec1c98 0x551555 0x5a9dac 0x50a433 0x50cc96 0x507be4 0x509900 0x50a2fd 0x50cc96 0x508cd5 0x594a01 0x59fd0e 0x5576d8 0x50c19e 0x507be4 0x508ec2 0x594a01 0x549e8f 0x5515c1 0x5a9dac 0x50a433 0x50beb4 0x507be4 0x508f37 0x594a01 0x549e8f 0x5515c1 0x5a9dac
/usr/local/lib/python3.6/dist-packages/pytorch3d/io/obj_io.py:457: UserWarning: Mtl file does not exist: /content/GIF/GIF_resources/input_files/flame_resource/template.mtl
  warnings.warn(f"Mtl file does not exist: {f}")
generate_random_samples.py:108: VisibleDeprecationWarning: Creating an ndarray from ragged nested sequences (which is a list-or-tuple of lists-or-tuples-or ndarrays with different lengths or shapes) is deprecated. If you meant to do this, you must specify 'dtype=object' when creating the ndarray
  np.random.uniform(0, np.pi / 12, 1), 0, 0]).astype('float32')
creating the FLAME Decoder
../my_utils/photometric_optimization/models/FLAME.py:81: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.register_buffer('dynamic_lmk_faces_idx', torch.tensor(lmk_embeddings['dynamic_lmk_faces_idx'], dtype=torch.long))
../my_utils/photometric_optimization/models/FLAME.py:82: UserWarning: To copy construct from a tensor, it is recommended to use sourceTensor.clone().detach() or sourceTensor.clone().detach().requires_grad_(True), rather than torch.tensor(sourceTensor).
  self.register_buffer('dynamic_lmk_bary_coords', torch.tensor(lmk_embeddings['dynamic_lmk_bary_coords'], dtype=self.dtype))
generator const_input n_params: 8192
generator to_rgb n_params: 1568667
generator progression n_params: 27955884
generator z_to_w n_params: 2101248
Generating_images:   0% 0/4 [00:00<?, ?it/s]/usr/local/lib/python3.6/dist-packages/torch/nn/functional.py:3121: UserWarning: Default upsampling behavior when mode=bilinear is changed to align_corners=False since 0.4.0. Please specify align_corners=True if the old behavior is desired. See the documentation of nn.Upsample for details.
  "See the documentation of nn.Upsample for details.".format(mode))
Generating_images: 100% 4/4 [00:19<00:00,  4.81s/it]
Saving images: 100% 128/128 [00:05<00:00, 24.41it/s]
Saving images: 100% 128/128 [00:00<00:00, 354.92it/s]

**Comment by ParthaEth** [comment]
I have no idea what this warning means. Hahaha. I always ignored it I think.
